setup.cfg

```diff
@@ -12,7 +12,32 @@ tag_prefix =
 parentdir_prefix = great_expectations-
 
 [flake8]
-max-line-length = 200
-exclude = tests/*,docs/*
+exclude = tests/*,
+    docs/*,
+    build/*,
+    assets/*,
+    contrib/*,
+    versioneer*,
+    examples/*,
+    # TODO: remove the items below and fix linting issues
+    great_expectations/checkpoint,
+    great_expectations/self_check,
+    great_expectations/jupyter_ux,
+    great_expectations/execution_engine,
+    great_expectations/profile,
+    great_expectations/cli,
+    great_expectations/expectations,
+    great_expectations/dataset,
+    great_expectations/datasource,
+    great_expectations/data_context,
+    great_expectations/marshmallow__shade,
+    great_expectations/render,
 per-file-ignores =
     */__init__.py: F401
+# E501 - line length (black)
+# E203 - whitespace before : (conflicts with black)
+# E402 - module level import not at top of file: (isort)
+extend-ignore = E501,E203,E402
+# https://github.com/pycqa/mccabe
+# TODO: reduce this 10
+max-complexity = 15
```

Comment:
This may warrant some discussion.

Reply:
Agreed. Could you ask this in our slack channel?